```
FAILED TEST SUMMARY
---------------------------------------------------------------------
Bug #66338 (Optimization binding of class constants is not safely opcacheable) [tests/bug66338.phpt]
Bug #78106: Path resolution fails if opcache disabled during request [tests/bug78106.phpt]
ISSUE #115 (path issue when using phar) [tests/issue0115.phpt]
ISSUE #149 (Phar mount points not working this OPcache enabled) [tests/issue0149.phpt]
revalidate_path 01: OPCache must cache only resolved real paths when revalidate_path is set [tests/revalidate_path_01.phpt]
=====================================================================
make: *** [Makefile:133: test] Error 1
error: builder for '/nix/store/niv042q5y701gz7jyjz16gcpgjn09793-php-opcache-8.1.15.drv' failed with exit code 2;
       last 10 log lines:
       > =====================================================================
       > FAILED TEST SUMMARY
       > ---------------------------------------------------------------------
       > Bug #66338 (Optimization binding of class constants is not safely opcacheable) [tests/bug66338.phpt]
       > Bug #78106: Path resolution fails if opcache disabled during request [tests/bug78106.phpt]
       > ISSUE #115 (path issue when using phar) [tests/issue0115.phpt]
       > ISSUE #149 (Phar mount points not working this OPcache enabled) [tests/issue0149.phpt]
       > revalidate_path 01: OPCache must cache only resolved real paths when revalidate_path is set [tests/revalidate_path_01.phpt]
       > =====================================================================
       > make: *** [Makefile:138: test] Error 1
       For full logs, run 'nix log /nix/store/niv042q5y701gz7jyjz16gcpgjn09793-php-opcache-8.1.15.drv'.
```

Amazing, those are the ones covered in #215004...

--- a/pkgs/top-level/php-packages.nix
+++ b/pkgs/top-level/php-packages.nix
@@ -84,7 +84,6 @@ lib.makeScope pkgs.newScope (self: with self; {
     , zendExtension ? false
     , doCheck ? true
     , extName ? name
-    , allowLocalNetworking ? false
     , ...
     }@args: stdenv.mkDerivation ((builtins.removeAttrs args [ "name" ]) // {
       pname = "php-${name}";
@@ -104,7 +103,6 @@ lib.makeScope pkgs.newScope (self: with self; {
       ];
 
       inherit configureFlags internalDeps buildInputs zendExtension doCheck;
-      __darwinAllowLocalNetworking = allowLocalNetworking;
 
       preConfigurePhases = [
         "cdToExtensionRootPhase"
@@ -413,8 +411,17 @@ lib.makeScope pkgs.newScope (self: with self; {
             valgrind.dev
           ];
           zendExtension = true;
+          postPatch = lib.optionalString stdenv.isDarwin ''
+            # Tests are flaky on darwin
+            rm ext/opcache/tests/blacklist.phpt
+            rm ext/opcache/tests/bug66338.phpt
+            rm ext/opcache/tests/bug78106.phpt
+            rm ext/opcache/tests/issue0115.phpt
+            rm ext/opcache/tests/issue0149.phpt
+            rm ext/opcache/tests/revalidate_path_01.phpt
+          '';
           # Tests launch the builtin webserver.
-          allowLocalNetworking = true;
+          __darwinAllowLocalNetworking = true;
         }
         {
           name = "openssl";

